src/ember-app.js

```diff
@@ -435,8 +435,6 @@ function createShoebox(doc, fastbootInfo) {
   if (!shoebox) { return; }
 
   for (let key in shoebox) {
-    if (!shoebox.hasOwnProperty(key)) { continue; }
```

Comment:
to future proof this. We could use `Object.hasOwnProperty(showbox, key)` instead?